article/RNH_AR_EG_SSTJ_EP_MMC-2016.md

```diff
@@ -146,89 +159,106 @@ of the jacobian derivation see supplementary_notebook_2.ipynb). As an expansion
-direction with diffusion weighting b-values of 500 and 1500 $s.mm^{-2}$ and with six b-value=0 images.
+directions with diffusion weighting b-values of 500 and 1500 $s.mm^{-2}$ and with six $b=0 s.mm^{-2}$ images.
 These simulations correspond to the results reported in Fig.5 of the original article.
 We conducted Monte Carlo simulations using the multi-tensor simulation
 module available in Dipy and using identical simulated acquisition parameters.
 As in the original article, fitting procedures are tested for voxels with five different FA values
 and with constant diffusion trace of $2.4 \times 10^{-3} mm^{2}.s^{-1}$.
 The eigenvalues used for the five FA levels are reported in Table @tbl:table.
 
-Table: Eigenvalues values used for the simulations {#tbl:table}
+Table: Eigenvalues used for the simulations {#tbl:table}
 
 FA            0                      0.11                   0.22                   0.3                    0.71
 ------------ ---------------------- ---------------------- ---------------------- ---------------------- ----------------------
 $\lambda_1$  $8.00 \times 10^{-4}$  $9.00 \times 10^{-4}$  $1.00 \times 10^{-3}$  $1.08 \times 10^{-3}$  $1.60 \times 10^{-3}$
 $\lambda_2$  $8.00 \times 10^{-4}$  $7.63 \times 10^{-4}$  $7.25 \times 10^{-4}$  $6.95 \times 10^{-4}$  $5.00 \times 10^{-4}$
 $\lambda_3$  $8.00 \times 10^{-4}$  $7.38 \times 10^{-4}$  $6.75 \times 10^{-4}$  $6.25 \times 10^{-4}$  $3.00 \times 10^{-4}$
 
-For each FA value, eleven different degrees of free water contamination were
-evaluated (f values equally spaced from 0 to 1). To assess the robustness of the
+For each FA value, 11 degrees of free water contamination were
+evaluated (f values were linearly spaced from 0 to 1). To assess the robustness of the
 procedure, Rician noise with signal-to-noise ratio (SNR) of 40 relative to the b-value = 0 images was
-used. For each FA and f-value pair, simulations were performed for 120
-different diffusion tensor orientation. Simulations for each diffusion tensor
-orientation were repeated 100 times making a total of 12000 simulation
+used. 
+
+stuff here
+
+For each FA and f-value pair, simulations were performed for 120
+different diffusion tensor orientations. Simulations for each diffusion tensor
+orientation were repeated 100 times for a total of 12 000 simulations
 iterations for each FA and f-value pair.
 
 ## In vivo data
 
-Similarly to the original article, the procedures are also tested using in vivo human brain data [@valabregue2015], that can be automatically
-downloaded by Dipy's functions. The original dataset consisted of 74 volumes of images acquired for a
-b-value of $0 s.mm^{-2}$ and 578 volumes diffusion weighted images acquired along 16 diffusion gradient directions
+Similarly to the original article, the procedures are also tested using in vivo human 
+brain data [@valabregue2015], that can be automatically
+downloaded by Dipy's functions.
+
+stuff here
```

Comment:
adding a quick note about the function module (dipy.data I'd say) would make it easier for people to fetch the stuff, or the original webpage if not.